src/Run/Interop.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.Execute
+{
+    internal class Interop
+    {
+        public static bool GetUnixVersion(out string result)
+        {
+            result = null;
+            const string OSReleaseFileName = @"/etc/os-release";
+            if (File.Exists(OSReleaseFileName))
+            {
+                string content = File.ReadAllText(OSReleaseFileName);
+                int idIndex = content.IndexOf("ID");
+                int versionIndex = content.IndexOf("VERSION_ID");
+                if (idIndex != -1 && versionIndex != -1)
+                {
+                    string id = content.Substring(idIndex + 3, content.IndexOf(Environment.NewLine, idIndex + 3) - idIndex - 3);
+                    string version = content.Substring(versionIndex + 12, content.IndexOf('"', versionIndex + 12) - versionIndex - 12);
```

Comment:
Is this assuming that VERSION_ID is always wrapped with quotes?  I don't believe that's always the case.  Here's the /etc/os-release from Fedora 23

```
[root@4f89a32f501b /]# cat /etc/os-release
NAME=Fedora
VERSION="23 (Twenty Three)"
ID=fedora
VERSION_ID=23
PRETTY_NAME="Fedora 23 (Twenty Three)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:23"
HOME_URL="https://fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=23
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=23
PRIVACY_POLICY_URL=https://fedoraproject.org/wiki/Legal:PrivacyPolicy
```